### PR TITLE
Add Elmax integration documentation.

### PR DESCRIPTION
**Pull request by albertogeniola**
## Proposed change

Adds Elmax documentation as requested by this PR: https://github.com/home-assistant/core/pull/59321

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/59321
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2941
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/elmax.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: ELMAX
+description: Instructions on how to integrate Elmax panels within Home Assistant.
+ha_category:
+  - Switch
+ha_release: 2021.12
```
Comment (marvin-w) [inline source/_integrations/elmax.markdown]:
```suggestion
ha_release: 2022.01
```

File: source/_integrations/elmax.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ha_iot_class: Cloud Poll
+ha_config_flow: true
+ha_quality_scale: internal
```
Comment (marvin-w) [inline source/_integrations/elmax.markdown]:
```suggestion
```
Elmax has no quality scale right now

**Review by marvin-w — CHANGES_REQUESTED** [review]

File: source/_integrations/elmax.markdown [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: ELMAX
+description: Instructions on how to integrate Elmax panels within Home Assistant.
+ha_category:
+  - Switch
+ha_release: 2022.01
```
Comment (marvin-w) [inline source/_integrations/elmax.markdown]: Sorry my suggestion was wrong! There will not be a 2022.1 release.

```suggestion
ha_release: 2022.2
```

Comment (albertogeniola) [reply source/_integrations/elmax.markdown]: Done!

**Review by marvin-w — DISMISSED** [review]
Looks good! Can be merged as soon as the parent PR is merged.

File: source/_integrations/elmax.markdown [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ha_iot_class: Cloud Poll
+ha_config_flow: true
+ha_codeowners:
+  - '@albertogeniola'
+ha_domain: elmax
+ha_platforms:
+  - switch
+---
+
+The `Elmax` integration enables Home Assistant integration with
```
Comment (frenck) [inline source/_integrations/elmax.markdown]:
```suggestion
The Elmax integration enables Home Assistant integration with 
```

File: source/_integrations/elmax.markdown [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ha_platforms:
+  - switch
+---
+
+The `Elmax` integration enables Home Assistant integration with 
+[ELMAX](https://www.elmaxsrl.it/) control panels. Elmax panels are 
+domotic systems focused on intrusion detection and home surveillance.
+
+The current integration is capable of controlling domotic switches 
+and actuators connected to the elmax panels.
```
Comment (frenck) [inline source/_integrations/elmax.markdown]:
```suggestion
The current integration is capable of controlling domotic switches 
and actuators connected to the Elmax panels.
```

File: source/_integrations/elmax.markdown [hunk 6]
```diff
@@ -0,0 +1,23 @@
+---
+title: ELMAX
```
Comment (frenck) [inline source/_integrations/elmax.markdown]:
```suggestion
title: Elmax
```

File: source/_integrations/elmax.markdown [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+description: Instructions on how to integrate Elmax panels within Home Assistant.
+ha_category:
+  - Switch
+ha_release: 2022.2
+ha_iot_class: Cloud Poll
```
Comment (frenck) [inline source/_integrations/elmax.markdown]:
```suggestion
ha_iot_class: Cloud Polling
```

**Review by frenck — DISMISSED** [review]

File: source/_integrations/elmax.markdown (hunk 7 above)
Comment (albertogeniola) [reply source/_integrations/elmax.markdown]: Done

File: source/_integrations/elmax.markdown (hunk 6 above)
Comment (albertogeniola) [reply source/_integrations/elmax.markdown]: Done

File: source/_integrations/elmax.markdown (hunk 5 above)
Comment (albertogeniola) [reply source/_integrations/elmax.markdown]: Done

File: source/_integrations/elmax.markdown (hunk 4 above)
Comment (albertogeniola) [reply source/_integrations/elmax.markdown]: Done

**Review by marvin-w — APPROVED** [review]
Looks good for me! Thanks @albertogeniola!

**Pull request merged**